### PR TITLE
Update PRRTE debugger examples

### PR DESCRIPTION
**Pull request by drwootton**
Update attach example to expect PMIX_ERR_LOST_CONNECTION event as notification

that run is complete.

Update debugger daemon to eliminate printfs following PMIX_tool_finalize since
forwarding of them to tool process is inconsistent.

Signed-off-by: David Wootton <dwootton@us.ibm.com>

### CODE REVIEW

**Comment by jjhursey** [comment]
@drwootton It looks like this branch needs to be rebased to resolve the conflicts with the current `master` branch.

File: examples/debugger/indirect.c [hunk 1]
```diff
@@ -60,8 +60,8 @@ static void notification_fn(size_t evhdlr_registration_id, pmix_status_t status,
     if (NULL != cbfunc) {
         cbfunc(PMIX_EVENT_ACTION_COMPLETE, NULL, 0, NULL, NULL, cbdata);
     }
-    ilactive = false;
     printf("\tCOMPLETE\n");
+sleep(2);
```
Comment (jjhursey) [inline examples/debugger/indirect.c:64]: Is the `sleep(2)` required here? If so can you add a comment above it as to why?

**Review by jjhursey — COMMENTED** [review]
This looks good. I had one comment. Will review it again after it is rebased.

**Comment by rhc54** [comment]
My apologies - I created the conflicts, so I fixed them here. Feel free to squash.

**Comment by jjhursey** [comment]
Closing. Replaced by PR #1030

**Pull request closed without merging**